pkgs/by-name/he/hexxy/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  gitUpdater,
+}: let
+  pname = "hexxy";
+  version = "unstable-2024-06-07";
```

```suggestion
  version = "0-unstable-2024-06-07";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming